src/test/java/ca/wolfram/beta/symath/VMapTest.java

```diff
@@ -44,11 +44,12 @@ public void getWithDefault() {
 
     @Test
     public void getWithError() {
+        String error = "var not defined";
         try {
-            map.get("a", "var not defined");
+            map.get("a", error);
```

Comment:
It's purely so that in the event I change the error, I don't have to do it 3 times. I try to reduce new String creations whenever possible since it might take a test build for me to realize I forgot something